src/cmds/configure.py

```diff
@@ -1,76 +1,24 @@
+import click
+from pathlib import Path
+
 from src.util.config import (
     load_config,
     save_config,
 )
-from argparse import ArgumentParser
 from typing import Dict
 from src.util.default_root import DEFAULT_ROOT_PATH
 from src.util.config import str2bool
 
 
-def make_parser(parser: ArgumentParser):
-
-    parser.add_argument(
-        "--set-node-introducer",
-        help="Set the introducer for node - IP:Port",
-        type=str,
-        nargs="?",
-        default="",
-    )
-
-    parser.add_argument(
-        "--set-fullnode-port",
-        help="Set the port to use for the fullnode",
-        type=str,
-        nargs="?",
-        default="",
-    )
-
-    parser.add_argument(
-        "--set-log-level",
-        "--log-level",
-        "-log-level",
-        help="Set the instance log level, Can be CRITICAL, ERROR, WARNING, INFO, DEBUG, NOTSET",
-        type=str,
-        nargs="?",
-        default="",
-    )
-
-    parser.add_argument(
-        "--enable-upnp",
-        "--upnp",
-        "-upnp",
-        help="Enable or disable uPnP. Can be True or False",
-        type=str,
-        nargs="?",
-    )
-
-    parser.set_defaults(function=configure)
-
-
-def help_message():
-    print("usage: chia configure -flag")
-    print(
-        """
-        chia configure [arguments] [inputs]
-            --set-node-introducer [IP:Port] (Set the introducer for node),
-            --set-fullnode-port [Port] (Set the full node default port, useful for beta testing),
-            --set-log-level [LogLevel] (Can be CRITICAL, ERROR, WARNING, INFO, DEBUG, NOTSET),
-            --enable-upnp,
-            --upnp {True,False} (Enable or disable uPnP. Can be True or False)
-        """
-    )
-
-
-def configure(args, parser):
+def configure(root_path: Path, set_node_introducer: str, set_fullnode_port: str, set_log_level: str, enable_upnp: str):
     config: Dict = load_config(DEFAULT_ROOT_PATH, "config.yaml")
```

Comment:
Should it load the existing configuration (looks like using always the root configuration as base for the changes).

Reply:
I'm not sure about that. If it's always using root, i think it's fine to leave it that way for now.

Reply:
Ok. My way of thinking about this was: "I want to change the log level, so I use `chia --root-path .my-custom-root-path configure --set-log-level DEBUG`, that should respect the rest of my configuration and change only the log level, but what it does is. read the `DEFAULT_ROOT_PATH` config, set the log level, and overwrite the `.my-custom-root-path/config.yaml`. So it can overwrite things that I changed before.

I don't have the whole context, so of course we can leave it as is now, and you can think what makes more sense.

If the approach is to configure everything in one shot, probably makes no sense to disallow to execute `--set-node-introducer` and `--set-fullnode-port`, because if you do that, you end up without having the posibility of set the full node port everywhere and change the node introducer host (using the CLI).

Reply:
This is usually a flurry of commands issued to configure before start out of terraform and into a docker.

Reply:
Ok, what about review and merge (if is ok) this PR as is now, and somebody from chia can rethink about the CLI commands and I can try to help writting the changes.